galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.resource.manager/src/main/java/dev/galasa/cicsts/resource/internal/CicsBundleImpl.java

```diff
@@ -406,11 +340,11 @@ public boolean isEnabled() throws CicsBundleResourceException {
 	public boolean disable() throws CicsBundleResourceException {
         try {
             if (!resourceInstalled()) {
-                throw new CicsJvmserverResourceException(RESOURCE_TYPE_BUNDLE + " " + getName() + " does not exist");
+                throw new CicsJvmserverResourceException(RESOURCE_TYPE_BUNDLE + " " + getDefinitionName() + " does not exist");
             }
-            this.cicsRegion.cemt().setResource(this.cicsTerminal, RESOURCE_TYPE_BUNDLE, getName(), "DISABLED");
+            this.cicsRegion.cemt().setResource(this.cicsTerminal, RESOURCE_TYPE_BUNDLE, getDefinitionName(), "DISABLED");

```

Comment:
Not sure what it does, but pls pardon the obvious question: Should "ENABLED" and "DISABLED" be symbolic constants rather than literals, so they can be re-used elsewhere and documented what values are allowed ? If it's only those two possible values, should it be a boolean instead of a string ?

Reply:
Not got to try it as there's no jvmserver support for my test bundles.  Possibly could be, and maybe more states.